.travis.yml

```diff
@@ -37,7 +37,7 @@ script:
         cargo rustc --verbose --example ci_test -- -Zno-trans &&
         cargo rustc --verbose --example key_value_store -- -Zno-trans &&
         cargo rustc --verbose -- -Zno-trans &&
-        cargo test  --verbose --features use-mock-crust
+        env RUSTFLAGS="-C codegen-units=8" cargo test --release --verbose --features use-mock-crust --test mock_crust_tests
```

Comment:
Oh yes, you're right! I forgot about the library tests inside `src`. I mainly added that line so that the examples wouldn't be compiled in release mode, as that causes `src/lib.rs` to be compiled twice. I'll add another line that just runs the library tests?